src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -685,6 +675,47 @@ private void getCanvasResponse(HttpServletRequest request,
 		out.flush();
 	}
 
```

Comment:
It would be helpful to add some comment sample of the `Link` header. So that it is easy to understand how this is extracted in the menthod